packages/core/src/provider/types.ts

```diff
@@ -128,6 +133,14 @@ export interface ISmartAccountProvider<
    */
   sendTransactions: (request: RpcTransactionRequest[]) => Promise<Hash>;
 
+  /**
+   * This takes contract deploy parameters and crafts a UserOperation to deploy the contract.
```

Comment:
This also sends the user operation right?

Reply:
yup! lemme update that